docs/Configuring-Orion/TLS.md

```diff
@@ -3,17 +3,160 @@ description: TLS
 
 # TLS
 
-Orion supports the TLS (Transport Layer Security) protocol to enable secure communications.
+Orion supports the Transport Layer Security (TLS) protocol to enable secure communications. TLS is used between Orion nodes only. TLS is not used between Besu and Orion nodes.
 
 Enable TLS by setting the `tls` property to `strict` in the [Orion configuration file](Configuration-File.md).  
 
-## Getting Started with TLS
+## Generating certificates using Orion
 
 To have Orion generate certificates, start the nodes with the TLS trust mode set to `insecure-no-validation` for `tlsclienttrust` 
 and `tlsservertrust` until the `tlsknownclients` and `tlsknownservers` files are populated. When the files are populated, 
-restart the nodes with the TLS trust mode set to `whitelist` for `tlsclienttrust` and `tlsservertrust`. 
+restart the nodes with the TLS trust mode set to `whitelist` for `tlsclienttrust` and `tlsservertrust`.
 
-## Configuring TLS 
+## Generating certificates using OpenSSL
+
+These procedures explain how to use [OpenSSL](https://www.openssl.org/source/) to generate certificates when the Common Name (CN) is either the [public DNS](#public-dns-is-cn) or an [IP address](#ip-address-is-cn). Before you begin, ensure OpenSSL is installed.
+
+### Public DNS is CN
+
+To use a public DNS as CN:
+
+#### Generating a CA certificate
+
+1. Generate a key file called `orion_ca.key`:
+
+    `openssl genrsa -out orion_ca.key 2048`
+  
+1. Generate a certificate authority (CA) certificate called `orion_ca.pem` that uses `orion_ca.key`:
+
+    `openssl req -x509 -new -nodes -key orion_ca.key -sha256 -days 1024 -out orion_ca.pem`
+
+#### Generating a new certificate for a node
+
+We recommend each node has its own certificate. To generate the certificate:
+
+1. Generate a key file called `orion_cer.key`:
+
+    `openssl genrsa -out orion_cer.key 2048`
+  
+1. Generate a certificate signing request (CSR) called `orion_cer.csr`:
+
+    `openssl req -new -key orion_cer.key -out orion_cer.csr`
+  
+1. Answer each prompt for information to be added to the certificate request. Ensure the
+value you specify for Common Name (CN) matches the host public DNS so the requests from the server are accepted. The name 
+is specified in the configuration file for the `nodeurl` and `clienturl` options too.
```

Comment:
The name is specified in the configuration file for the `nodeurl` and `clienturl` options too.

I'd probably write this as -> The name is also specified in the configuration file for the `nodeurl` and `clienturl` options.

That's personal preference though so up to you.